recipes/util-log/codemod.yaml

```
@@ -0,0 +1,21 @@
+schema_version: "1.0"
+name: "@nodejs/util-log"
```

```suggestion
name: "@nodejs/util-log-to-console-log"
```

Is it better ? cc @JakobJingleheimer